main.py

```diff
@@ -26,22 +26,37 @@ def handle_error(e):
 @app.route("/", methods=["GET", "POST"])
 def upload():
     if request.method == "POST":
-        csvfile = request.files["csvfile"]
-        # Do something with the uploaded CSV file...
-        csv_data = csvfile.stream.read().decode("utf-8")
-        samplesheet = generate_genomics_sheet(csv_data, request.form)
+        # normal project
```

Comment:
the try / except is because with raw data we need to POST but we are not uploading any files, the rawdata looks like this:
```
[Header]
FileFormatVersion,1,
DevelopmentProject,No,
Flowcell,AACHWK,
Project_ID,2023_174,
```

Reply:
I made a new commit where I specify the exception, it is coming from pandas because pandas will try to read the TextIO objects and they are empty, so when they are empty it will check if this is raw data and that that has been filled out correctly otherwise it will throw the appropriate unhandled exception